### ISSUE TITLE
marsjaninzmarsa → "dodawanie issue z mojego konta a nie z konta Alicjowego, po wcześniejszej autoryzacji"

### ISSUE THREAD

**Issue by MargatroidAlice** [TeMPOraL/alice#176]

**Comment by marsjaninzmarsa** [comment]
Pracuję nad tą kwestią przy okazji mojej własnej planowanej integracji z wiadomym API (ale cicho sza!), generalnie Oauth2 + Person + Persistence.

Sam tego nie ogarnę chyba w skończonym czasie, ale jak się chcesz tym zająć to raczej przyda się to obu projektom. :)

**Comment by TeMPOraL** [comment]
Ok, z tego się robi ogólne ogarnianie okołoOAuthowych i autoryzacyjnych rzeczy.

**Comment by marsjaninzmarsa** [comment]
Tak, jak już to będzie to przepięcie odpowiedniego tokena w zależności od proszącego będzie formalnością.
